### PR TITLE
Include script to render lidarseg histogram for split

### PR DESCRIPTION
**Pull request by whyekit-aptiv**
### This PR will:
- Introduce a script to enable users to render a histogram for a given nuScenes split

### CODE REVIEW

File: python-sdk/nuscenes/lidarseg/lidar_dataviz.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        plt.tight_layout()
+        fig.savefig(save_as_img_name)
+
+    if verbose:
+        plt.show()
+
+
+def get_lidarseg_stats(nusc: NuScenes, sort_by: str = 'count_desc') -> Tuple[List[str], List[int]]:
+    """
+    Get the number of points belonging to each class for the given nusc split.
```
Comment (holger-nutonomy) [inline python-sdk/nuscenes/lidarseg/lidar_dataviz.py:152]: nuScenes split

File: python-sdk/nuscenes/lidarseg/lidar_dataviz.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    :param nusc: A nuScenes object.
```
Comment (holger-nutonomy) [inline python-sdk/nuscenes/lidarseg/lidar_dataviz.py:153]: NuScenes

File: setup/requirements.txt [hunk 3]
```diff
@@ -11,6 +11,7 @@ Pillow<=6.2.1  # Latest Pillow is incompatible with current torchvision, https:/
 pyquaternion>=0.9.5
 scikit-learn
 scipy
+seaborn
```
Comment (holger-nutonomy) [inline setup/requirements.txt:14]: Let's add a comment instead in the visualization script.

File: python-sdk/nuscenes/lidarseg/lidar_dataviz.py [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+from matplotlib.ticker import FuncFormatter, ScalarFormatter
+import matplotlib.transforms as mtrans
+import numpy as np
+import pandas as pd
+import seaborn as sns
+
+from nuscenes import NuScenes
+from nuscenes.utils.color_map import get_colormap
+
+
```
Comment (holger-nutonomy) [inline python-sdk/nuscenes/lidarseg/lidar_dataviz.py:15]: Suggest to rename the file to class_histogram.py

File: python-sdk/nuscenes/lidarseg/lidar_dataviz.py [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+    start_time = time.time()
+
+    # Get the statistics for the given nuScenes split.
+    class_names, counts = get_lidarseg_stats(nusc, sort_by=sort_by)
+
+    print('Calculated stats for {} point clouds in {:.1f} seconds.\n====='.format(
+        len(nusc.lidarseg), time.time() - start_time))
+
+    # Place the class names and counts into a dataframe for use with seaborn later.
+    df = pd.DataFrame(list(zip(class_names, counts)), columns=['Class', 'Count'])
```
Comment (holger-nutonomy) [inline python-sdk/nuscenes/lidarseg/lidar_dataviz.py:97]: Let's not use pandas here, as it is not a base requirement

File: setup/requirements.txt (hunk 3 above)
Comment (whyekit-aptiv) [reply setup/requirements.txt]: Changed to plotting using matplotlib instead of seaborn

File: python-sdk/nuscenes/lidarseg/lidar_dataviz.py (hunk 5 above)
Comment (whyekit-aptiv) [reply python-sdk/nuscenes/lidarseg/lidar_dataviz.py:97]: Found workaround to avoid using pandas

**Pull request merged**